### PR TITLE
User input box

### PR DESCRIPTION
**Pull request by jaypatel31**
## Describe your changes
Created a box for the user to input their value and get them converted to the desired unit.

## Issue number and link
#4 

## Screenshots (if Applicable)
Screenshot 2023-11-24 at 12 07 23 PM
Screenshot 2023-11-24 at 12 07 43 PM

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature
- [x] I have added thorough tests

### CODE REVIEW

File: code/popup/popup.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <div class="unit">
+              <input type="number" id="right_input" name="left_input" value="0"/>
+              <select id="unit_selector_right" class="selector" name="right_unit">
+                <option>Please Select Type</option>
+              </select>
+            </div>
+          </div>
+      </div>
+
+          <!-- <li style="padding-top: 10px">
```
Comment (drs1951) [inline code/popup/popup.html:41]: Remove commented code.

File: code/conversions/conversion.js [hunk 2]
```diff
@@ -6,10 +6,11 @@ const getPrecisionV = (number) => {
   let parts = number.toString().split(".");
   let intlen = number < 0 ? parts[0].length - 1 : parts[0].length;
   return intlen + parts[1].length;
+
 }
 
 const getPreciseNumberV = (number, precision) => {
-  if (number < 10e-3) {
+  if (number < 0) {
```
Comment (drs1951) [inline code/conversions/conversion.js:13]: Why has this line been changed?

File: code/popup/popup.html (hunk 1 above)
Comment (drs1951) [inline code/popup/popup.html:41]: Remove commented code.

Comment (jaypatel31) [reply code/popup/popup.html]: This is removed

File: code/conversions/conversion.js (hunk 2 above)
Comment (jaypatel31) [reply code/conversions/conversion.js:13]: So whenever 0 value get inserted, the code breaks due to this line and function call to getPreciseNumber

Comment (drs1951) [reply code/conversions/conversion.js:13]: Got it. Looks good.

**Review by drs1951 — APPROVED** [review]

**Pull request merged**